fv/winfv/policy_test.go

```diff
@@ -164,6 +164,10 @@ var _ = Describe("Windows policy test", func() {
 			// Assert that it's now reachable.
 			err = kubectlExec(fmt.Sprintf(`-t porter -- powershell -Command 'Invoke-WebRequest -UseBasicParsing -TimeoutSec 5 %v'`, nginxB))
 			Expect(err).NotTo(HaveOccurred())
+
+			// Remove resources
+			_, err = client.NetworkPolicies().Delete(context.Background(), "demo", "allow-nginx-b", options.DeleteOptions{})
```

Comment:
Should do cleanup in an AfterEach or a `defer` to avoid test leakage if the test fails.